src/hind_brain/hind_brain.ino

```diff
@@ -29,11 +29,14 @@ boolean isAuto = false;
 unsigned int velMsg = VEL_CMD_MIN;        // Initialize velocity to 0
 signed int steerMsg = STEER_CMD_CENTER;   // Initialize steering to straight
 char buf[7];
+long watchdog_timer;
+
```

Comment:
Assuming the long is 32 bits, the tractor can run for almost 25 days without overflowing. However, we might as well leave the option of unlimited moving open, as it would only require adding a counter reset logic.

Reply:
If you want to take this up another power of 2, an unsigned long will also do the job, but I would go with the above suggestion.

Reply:
Turns out that millis() is an unsigned long, since we cannot do anything about that, the best you can do is change long to unsigned long here.